test/jdk/java/lang/Thread/jni/AttachCurrentThread/ImplicitAttach.java

```diff
@@ -62,7 +62,7 @@ public static void main(String[] args) throws Throwable {
 
         // void start_threads(int count, void *(*f)(void *))
         SymbolLookup symbolLookup = SymbolLookup.loaderLookup();
-        MemorySegment symbol = symbolLookup.lookup("start_threads").orElseThrow();
+        MemorySegment symbol = symbolLookup.find("start_threads").orElseThrow();
```

Comment:
This is dealt with in another PR: https://github.com/openjdk/panama-foreign/pull/722